scrapy/utils/url.py

```diff
@@ -103,3 +103,16 @@ def guess_scheme(url):
         return any_to_uri(url)
     else:
         return add_http_if_no_scheme(url)
+
+
+def trim_href_attribute(href):
```

Comment:
Whitespaces should be stripped form `<img src>` values as well - https://www.w3.org/TR/html5/embedded-content-0.html#attr-img-src. So `trim_href_...` is not a good name indeed.